pg_search/src/postgres/customscan/pdbscan/mod.rs

```diff
@@ -563,6 +563,21 @@ impl CustomScan for PdbScan {
         mut builder: CustomScanStateBuilder<Self, Self::PrivateData>,
     ) -> *mut CustomScanStateWrapper<Self> {
         unsafe {
+            let rti_list = (*builder.args().cscan).custom_relids;
+            let mut rti_list_iter = bms_iter(rti_list);
+
+            // Note: the range table index at execution time might be different from the one at planning time,
+            // so we need to use the one at execution time when creating the custom scan state.
```

Comment:
This comment should have the doc reference that is currently near the other assertion, I think.